src/components/CustomRouter/CustomRoute/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Route, Redirect } from 'react-router-dom';
+
+import { PATHS } from 'constants/paths';
+
+interface Props {
+  isPrivate?: boolean;
+  path: string;
+  screen: (props: Record<string, unknown>) => JSX.Element;
```

Comment:
Since `<Route>` already recognizes these props this interface can extend from the types defined by "react-router-dom", so you wouldn't need to type them and you could pass them with the spread operator.